pages/docs/reference/coding-conventions.md

```diff
@@ -726,6 +727,242 @@ foo {
 
 </div>
 
+### Trailing commas
+
+A trailing comma is a comma sign after the last item of the element series:
+
+<div class="sample" markdown="1" theme="idea" mode="kotlin" data-highlight-only>
+
+```kotlin
+class Person(
+    val name: String,
+    val surname: String,
+    val age: Int, // trailing comma
+)
+```
+
+</div>
+
+Using trailing commas has several benefits:
+
+* It makes version-control diffs cleaner: all focus is on changed value.
+* It's easy to add or reorder elements: no need to add or delete the comma sign if you manipulate elements.
+* It simplifies code generation (for example, for object initializers). The last element can also have the comma sign.
+
+Using trailing commas are optional: your code will work without them. But Kotlin style guide encourages the use of trailing commas at the declaration site and leaves it optional for the call site.
```

Comment:
```suggestion
Trailing commas are optional: your code will work without them. Kotlin style guide encourages the use of trailing commas at the declaration site and leaves it optional for the call site.
```
- With `Using`, the 'are' verb is incorrect.
- `But` doesn't seem necessary and starting a sentence with it is an arguable topic, so I'd just remove it.
- `optional` is used in two different meanings in one line. We should replace it with smth that conducts `up to you` meaning better.